### PR TITLE
Two step checkout page 2 (checkout) layout/style amends

### PR DESCRIPTION
**Pull request by rBangay**
## What are you doing in this PR?
Layout and style amends to the second page of the two step checkout.

Currently behind the `twoStepCheckout` ab test here ->
[https://support.thegulocal.com/uk/contribute#ab-twoStepCheckout=variant](https://support.thegulocal.com/uk/contribute#ab-twoStepCheckout=variant)

[***Trello Card**](https://trello.com/c/VELeJraK/1493-2-step-checkout-implement-design-updates-to-page-2-of-2-step-checkout)

#### single contribution
Before (AB variant)             |  After (AB variant)
:-------------------------:|:-------------------------:
  |  

#### recurring contribution
Before (AB variant)             |  After (AB variant)
:-------------------------:|:-------------------------:
  |

### CODE REVIEW

File: support-frontend/assets/components/personalDetails/personalDetails.tsx [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
+
 const hiddenHeading = css`
 	${visuallyHidden};
 `;
 
-const fieldGroupStyles = css`
+const fieldGroupStyles = (hideDetailsHeading?: boolean) => css`
 	position: relative;
 	margin-top: ${space[4]}px;
+	margin-top: ${hideDetailsHeading ? `${space[4]}px` : '0'};
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/components/personalDetails/personalDetails.tsx:30]: Is the margin-top needed on line above this one?

File: support-frontend/assets/components/personalDetails/personalDetails.tsx [hunk 2]
```diff
@@ -35,6 +50,8 @@ export type PersonalDetailsProps = {
 	onLastNameChange: (lastName: string) => void;
 	signOutLink: React.ReactNode;
 	contributionState: React.ReactNode;
+	overrideHeadingCopy?: string;
+	hideDetailsHeading?: true;
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/components/personalDetails/personalDetails.tsx:54]: Does this always default to hiding even if false supplied? Don't suppose this situation is likely?

File: support-frontend/assets/components/personalDetails/personalDetails.tsx (hunk 1 above)
Comment (rBangay) [reply support-frontend/assets/components/personalDetails/personalDetails.tsx:30]: good spot, I should have removed the top one ...

**Review by paul-daniel-dempsey — APPROVED** [review]
Looking great, matches PR screenshots nicely, thanks

**Pull request merged**